s3/variables.tf

```diff
@@ -85,6 +85,11 @@ variable "sns_notification" {
   default     = false
 }
 
+variable "lambda_notification" {
```

Comment:
Could this name and description be more specific to make it clear this is for the bulk upload of users?

Reply:
It doesn't explicitly have to be for the bulk upload of users, it could be for anything in the future where we want an s3 notification to go straight to a lambda. I'll put a description in to say that.